### ISSUE TITLE
The batch setting is the same, and the AP obtained by single-gpu and multi-gpu training is greatly different

### ISSUE THREAD

**Issue by gjxgjxgjxgjx** [open-mmlab/mmdetection#3621]
When I train on a single GPU, I set imgs_per_gpu = 8, four GPU training, set imgs_per_gpu = 2, but the accuracy of the model obtained is quite different, single gpu got 0.75, multi-gpu only got 0.71, tried several times.

**Comment by xvjiarui** [comment]
Hi @gjxgjxgjxgjx 
When you are using `imgs_per_gpu = 8`, the effective batch size of batch norm is 8. However, when `imgs_per_gpu = 2`, the effective batch size of batch norm will be 2. You may try to use SyncBN instead.

**Comment by gjxgjxgjxgjx** [comment]
@xvjiarui Thank you for your reply. After adopting SyncBN, can I get similar results by setting imgs_per_gpu = 2 for 4 gpus and imgs_per_gpu = 8 for a single gpu?

**Comment by xvjiarui** [comment]
Yes.
